app/services/eyeshade/publishers.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Eyeshade::Publishers < Eyeshade::BaseApiClient
```

Comment:
IMO We should rename this to `Eyeshade::Settlements`. Thoughts?

Reply:
I think we should keep it as a publishers because that's what the actual resource on eyeshade is. I imagined that there would be other things under the same uri structure 🤔